src/masonite/configuration/Configuration.py

```diff
@@ -63,6 +63,9 @@ def merge_with(self, path, external_config):
     def set(self, path, value):
         self._config[path] = value
 
+    def has(self, path):
+        return path in self._config[path]
```

Comment:
Is _config a dictionary? Won't this throw an exception if path is not in _config? Shouldn't this be `path in self._config.get(path)`

Reply:
My bad it should be : `return path in self._config`  `_config` is a dotty dict accessible with dots.